cloudify-manager-worker/CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
```

Comment:
I love the idea of the changelog!
How detailed do you want it to be? because if we want to specify all changes in every version from now on, let's also include:
Changed: ReadMe update
Added: Changelog
Fixed: cert-issuer default duration for new certs was 90 days - defaulted to 10y [editable]
Changed: postgres will create PV by default

What do you think? is it too much?

Reply:
Added more details to changeling.